packages/@react-spectrum/table/docs/TableView.mdx

```diff
@@ -1587,6 +1587,117 @@ function CustomDragPreviewExample() {
 <CustomDragPreviewExample />
 ```
 
+## Expandable Rows <VersionBadge version="alpha" style={{marginLeft: 4, verticalAlign: 'bottom'}} />
+
+By default, TableView doesn't allow expandable rows, but this can be enabled using the `hasExpandableRows` prop.
+You'll first need to enable a feature flag in your app to use this feature.
+
+```tsx example
+import {enableTableNestedRows} from '@react-stately/flags';
+
+enableTableNestedRows();
+```
+<br />
+
+Use `defaultExpandedKeys` to provide a default set of expanded rows. Note that the value of the expanded keys must match the `key` prop of the `Row` and that an
+expanded child row will only be visible if its parent row key is also included within the expanded key set.
+
+An expandable row can be statically defined by including one or more `Rows` as children. Note that all child `Rows` must be placed after the last `Cell` within the parent `Row`.
+
+The example below enables expandable rows, and uses `defaultSelectedKeys` to expand rows with keys "1" and "2".
```

Comment:
If this were its own component we'd have a separate example for `defaultSelectedKeys`. This is probably okay.

We aren't covering controlled `selectedKeys`.

Reply:
This should actually read `defaultExpandedKeys`, good catch